src/leappto/cli.py

```diff
@@ -387,14 +392,17 @@ def _virt_tar_out():
         def check_target(self):
             storage_dir = MACROCONTAINER_STORAGE_DIR
             ps_containers = 'docker ps -a --format "{{.Names}}"'
-            rc, containers = self._ssh_sudo_out(ps_containers)
+            rc, containers = self._ssh_sudo_out(ps_containers,
+                                                machine_context=self.TARGET)
             if rc:
                 return rc, []
             names = containers.splitlines()
             ls_storage_directories = 'ls -1 "{}"'.format(storage_dir)
-            rc, dirs = self._ssh_sudo_out(ls_storage_directories)
-            if not rc:
-                names.extend(dirs.splitlines())
+            rc, dirs = self._ssh_sudo_out(ls_storage_directories,
+                                          machine_context=self.TARGET)
+            if rc:
```

Comment:
Why did you revert this? 
It needs to stay:
```python
if not rc:
    names.extend(dirs.splitlines())
```

because otherwise you fail when the directory is missing on the target which is valid, the target directory is enough to be created on the migrate-machine call itself